doc/features/metrics/app-metrics/README.md

```diff
@@ -65,14 +65,14 @@ var cluster = Cluster
     .WithSocketOptions(new SocketOptions().SetReadTimeoutMillis(10000))
     .WithMetrics(metricsRoot.CreateDriverMetricsProvider(
         new DriverAppMetricsOptions()
-            .SetHighestLatencyMilliseconds(12000) // should be set to slightly higher than the configured timeout
+            .SetHighestLatencyMilliseconds(15000) // should be set to a value that is higher than the configured timeout
```

Comment:
Probably meant `30000` :)

Reply:
Hmm this is just an example on the manual, you think we should use the default here? I mention a bit later that the default is `30000`

Reply:
Yeah I think it's better actually since we were having log messages when the value wasn't high enough, I'll change it